packages/matchbox/src/components/ListBox/Option.js

```diff
@@ -21,7 +26,7 @@ const Option = React.forwardRef(function Option(props, ref) {
       aria-posinset={index}
       aria-selected={isActive}
       disabled={disabled}
-      onClick={() => onSelect(value)}
+      onClick={handleClick}
     >
       <StyledLink active={isActive} as="button" disabled={disabled} tabIndex="-1" ref={ref}>
```

Comment:
I _think_ the issue might be because this button doesn't have `type="button"`, since buttons default to submit

Reply:
Oh good call